src/pages/en/guides/deploy/github.md

```diff
@@ -15,15 +15,27 @@ Astro maintains the official `withastro/action` to deploy your project with very
 
 1. Set the [`site`](/en/reference/configuration-reference/#site) and, if needed, [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
     - `site` should be something like `https://<YOUR_USERNAME>.github.io`
-    - `base` should be your repository’s name starting with a forward slash, for example `/my-repo`.
+    - `base` should be your repository’s name starting with a forward slash, for example `/my-repo`. This is so that Astro understands your website's root is `/my-repo`, rather than the default `/`.
     
     :::note
-    If your repository is named `<YOUR_USERNAME>.github.io`, you don’t need to include `base`.
+    Don't set a `base` parameter if:
+
+    - your repository is named `<YOUR_USERNAME>.github.io`
+    - you're using a custom domain
```

Comment:
We tend to go with the AP Stylebook recommendation for bullet points, I would change this for consistency:
```suggestion
    - Your repository is named `<YOUR_USERNAME>.github.io`.
    - You're using a custom domain.
```